sphinx_gallery/scrapers.py

```diff
@@ -273,7 +273,9 @@ def _anim_rst(anim, image_path, gallery_conf):
     dpi = rcParams["savefig.dpi"]
     if dpi == "figure":
         dpi = fig.dpi
-    video_uri = video.relative_to(gallery_conf["src_dir"]).as_posix()
+    # relative_to doesn't work on windows
+    # video_uri = video.relative_to(gallery_conf["src_dir"]).as_posix()
+    video_uri = os.path.relpath(video, gallery_conf["src_dir"])
```

Comment:
I wouldn't expect this to work. This will have backslashes and URLs in HTML like the one in `video` below should have forward slashes, no? So there needs to be some posix-izing of the path.